nixos/modules/services/security/kanidm.nix

```diff
@@ -225,7 +228,8 @@ in
           bindaddress = mkOption {
             description = "Address/port combination the webserver binds to.";
             example = "[::1]:8443";
-            type = types.str;
+            default = null;
```

Comment:
I think it would be nicer to just repeat the upstream default in our default in nix. Then there won't be any confusion as to what the default actually will turn out to be and it allows users to refer to `config.kanidm.serverSettings.bindaddress` in other configuration files (e.g. nginx).

Reply:
Yeah I wasn't sure what the best practices for this are, whether to repeat upstream or let it default. I guess since we default the `serverPort` anyway it doesn't really matter.

Reply:
Adding to that point. We could also split the config into `bindHost` and `bindPort` which would relieve us of the string matching to get the port, but pretty sure that would be a breaking change.

Reply:
Might be a good idea, have to think about this a bit. But since this is breaking I think we should discuss this as a separate PR.

Reply:
Sounds good. I've choosen to copy the upstream config to make it more easily accessible in nix for now, but that still won't break existing setups.